packages/genesys-spark-components/src/components/beta/gux-avatar/example.html

```diff
@@ -470,121 +470,149 @@ <h2>Notifications</h2>
 >
 </gux-avatar-beta> -->
 
-<h2>Wrapped Avatar</h2>
+<h2>gux-avatar-focusable</h2>
 <p>
   The following shows examples of correct focus styling for the avatar when it
   is located within a focusable element such as a button or anchor tag.
 </p>
 <h3>Button</h3>
 <div class="wrapped-examples">
-  <button class="gux-avatar-button">
-    <gux-avatar-beta name="Conor Darcy"></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button">
-    <gux-avatar-beta
-      name="Conor Darcy"
-      presence="available"
-      badge
-    ></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button">
-    <gux-avatar-beta
-      name="Conor Darcy"
-      presence="available"
-      ring
-    ></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button">
-    <gux-avatar-beta
-      name="Conor Darcy"
-      presence="available"
-      badge
-      ring
-    ></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button">
-    <gux-avatar-beta
-      size="medium"
-      name="Conor Darcy"
-      presence="available"
-      badge
-      ring
-    ></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button gux-focus-ring-small-focused-visible">
-    <gux-avatar-beta
-      size="small"
-      name="Conor Darcy"
-      presence="available"
-      badge
-      ring
-    ></gux-avatar-beta>
-  </button>
-  <button class="gux-avatar-button gux-focus-ring-small-focused-visible">
-    <gux-avatar-beta
-      size="xsmall"
-      name="Conor Darcy"
-      presence="available"
-      badge
-      ring
-    ></gux-avatar-beta>
-  </button>
+  <gux-avatar-focusable-beta>
```

Comment:
I don't know if it's better to have this component within avatar folder and only have this example or to have a separate set of examples etc for the focusable avatar.